proto/desmos/subspaces/v1/subspace.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package desmos.subspaces.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/desmos-labs/desmos/v2/x/subspaces/types";
+
+// Subspace contains all the data of a Desmos subspace
+message Subspace {
+  option (gogoproto.equal) = true;
+  option (gogoproto.goproto_stringer) = true;
+
+  // Unique SHA-256 string that identifies the subspace
+  uint64 id = 1
+      [ (gogoproto.customname) = "ID", (gogoproto.moretags) = "yaml:\"id\"" ];
+
+  // Human-readable name of the subspace
+  string name = 2 [ (gogoproto.moretags) = "yaml:\"name\"" ];
+
+  // Optional description of this subspace
+  string description = 3 [ (gogoproto.moretags) = "yaml:\"description\"" ];
+
+  // Represents the account that is associated with the subspace and
+  // should be used to connect external applications to verify this subspace
+  string treasury = 4 [ (gogoproto.moretags) = "yaml:\"treasury\"" ];
+
+  // Address of the user that owns the subspace
+  string owner = 5 [ (gogoproto.moretags) = "yaml:\"owner\"" ];
+
+  // Address of the subspace creator
+  string creator = 6 [ (gogoproto.moretags) = "yaml:\"creator\"" ];
+
+  // the creation time of the subspace
+  google.protobuf.Timestamp creation_time = 7 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdtime) = true,
+    (gogoproto.moretags) = "yaml:\"creation_time\""
+  ];
```

Comment:
It should be useful to also have a `last_edited_time`. What do you think? @RiccardoM

Reply:
Agree with it. But it sounds costly if all the changes to the subspace including `Permissions` and `Group` related operation will update `last_update_date` field.

Reply:
Honestly I'm not very supportive of this idea. I don't see how a `last_edited` field would be useful for a subspace. It's not something that will be edited very often, and even if is it will be edited with very small changes. The last edited time can always be retrieved by looking at the transaction history (which also gives you the editor that applied the changes)

Reply:
Ok I see.